src/components/PageTitle.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <h1
+    class="text-[40px] font-[400] font-normal leading-[48px] color-[#34495e] not-italic">
```

Comment:
Color should be moved to tailwind config, but ignore it for now, since it's gonna be done in a different issue